packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -201,8 +207,11 @@ export const GiantTrackTile = ({
   const { isEnabled: isRemixContestEnabled } = useFeatureFlag(
     FeatureFlags.REMIX_CONTEST
   )
-
-  const isRemixContest = isRemixContestEnabled && !isOwner
+  const { data: events } = useEventsByEntityId(trackId, {
+    entityType: GetEntityEventsEntityTypeEnum.Track
+  })
+  const event = events?.[0]
```

Comment:
gross but didn't wanna bake in the assumption to always use the first event in the react-query hook?

Reply:
true its also possible that there are multiple events for a given entity id